Symfony/CS/FixerInterface.php

```diff
@@ -75,6 +75,8 @@ public function getPriority();
     /**
      * Returns true if the file is supported by this fixer.
      *
+     * @param \SplFileInfo $file A \SplFileInfo instance
```

Comment:
@both; thanks for the insight, makes sense :+1:  I'll remove the overcomplete `@param` references tomorrow.